java/core/src/test/org/apache/orc/TestColumnStatistics.java

```diff
@@ -28,6 +27,7 @@
 import org.apache.hadoop.hive.serde2.io.DateWritable;
 import org.apache.hadoop.hive.serde2.io.HiveDecimalWritable;
 import org.apache.hadoop.io.Text;
+import org.apache.hadoop.shaded.org.apache.commons.text.StringEscapeUtils;
```

Comment:
Is this change required?

Reply:
@wgtmac StringEscapeUtils is deprecated. I am split on this change, hence left a note in the PR description above, as commons-text is not available as direct dependency in ORC, and not inclined to add it just for one class.
Open to suggestions

Reply:
@dongjoon-hyun @williamhyun Any suggestion? I am not that familiar with that dependency.

Reply:
`*.shaded.` is supposed to be used by the original project only. Depending on other project's shaded dependency is worse than using a deprecated dependency. I agree with @wgtmac 's concern. Let's remove this.

Reply:
@dongjoon-hyun Ack, let me remove this change

Reply:
Rolled back the change, and updated PR description and JIRA to reflect the same, to avoid confusion in future

Reply:
@dongjoon-hyun Thanks for the feedback. Updated the description. Does it look better?